spiketools/sim/train.py

```diff
@@ -21,40 +21,39 @@ def sim_spiketrain(spike_param, n_samples, method, refractory=None, **kwargs):
         The number of samples to simulate.
     method : {'prob', 'binom', 'poisson'}
         The method to use for the simulation.
-    refractory : float, optional
-        The refractory period to apply to the simulated data, in seconds.
+    refractory : int, optional
```

Comment:
I found it less clear to use the `refractory` as an `int`.

Reply:
Yeh, this change here makes the refractory period definition more directly related to the spike representation, but makes it a little less intuitive for how we usually think about defining refractory periods. 

Part of the reason I changed this is that that defining this in terms of time means having to compute and potentially round the time into the number of samples, and how to round (up or down) might not be clear. In the previous version, there was actually a bug that would down to round down and not apply refractory periods at all in some cases - so I think defining spike train refractory periods in terms of number of samples is useful here in order to match the representation and remove some of the potential ambiguity.

Reply:
Ohh you're right! Sounds good then